Formula/rio.rb

```diff
@@ -0,0 +1,27 @@
+class Rio < Formula
+  desc "Hardware-accelerated GPU terminal emulator powered by WebGPU"
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "pkg-config" => :build
```

Comment:
Not really clear on why `pkg-config` is still needed.

Reply:
It is required by linux, but I think might not be necessary in macos. Let me try build without it.